libs/blocks/mnemonic-list/mnemonic-list.js

```diff
@@ -17,7 +17,13 @@ export const decorateMnemonicList = (container) => {
     if (title) product.appendChild(title);
     productList.appendChild(product);
     paragraph.replaceWith(productList);
+    paragraph.remove();
   });
+  if (targetElement) {
+    targetElement.prepend(productList);
+  }
```

Comment:
You could go for optional chaining and make it a one-liner, `targetElement?.prepend(productList);`

Reply:
Thanks! Fixed.